config/quality/checkstyle/checkstyle.xml

```diff
@@ -1,238 +1,240 @@
+        <module name="HiddenField" >
+            <property name="severity" value="ignore" />
+            </module>
+        <module name="IllegalCatch" />
+        <module name="IllegalInstantiation" />
+        <module name="IllegalThrows" />
+        <module name="IllegalToken">
+            <property name="tokens" value="LITERAL_NATIVE" />
+        </module>
+        <module name="IllegalTokenText">
+            <property name="tokens" value="STRING_LITERAL" />
+            <property name="format" value="a href" />
+        </module>
+        <module name="InnerAssignment" />
+        <module name="MagicNumber" />
+        <module name="MissingCtor" />
+        <module name="MissingSwitchDefault" />
+        <module name="ModifiedControlVariable" />
+        <module name="MultipleStringLiterals" />
+        <module name="MultipleVariableDeclarations" />
+        <module name="NestedForDepth" />
+        <module name="NestedIfDepth" />
+        <module name="NestedTryDepth" />
+        <module name="NoClone" />
+        <module name="NoFinalizer" />
+        <module name="OneStatementPerLine" />
+        <module name="OverloadMethodsDeclarationOrder" />
+        <module name="PackageDeclaration" />
+        <module name="ParameterAssignment" />
+        <module name="RequireThis" />
+        <module name="ReturnCount" />
+        <module name="SimplifyBooleanExpression" />
+        <module name="SimplifyBooleanReturn" />
+        <module name="StringLiteralEquality" />
+        <module name="SuperClone" />
+        <module name="SuperFinalize" />
+        <module name="UnnecessaryParentheses" />
+        <module name="VariableDeclarationUsageDistance" />
 
-    <module name="TreeWalker">
+        <!-- Checks for imports                              -->
+        <!-- See http://checkstyle.sourceforge.net/config_import.html -->
+        <module name="AvoidStarImport" />
+        <!-- We suppress this warning because the Java creators themselves disagree on this one, for example in the newest JUnit 5 -->
+        <module name="AvoidStaticImport">
+            <property name="severity" value="ignore" />
+        </module>
+        <module name="IllegalImport" />
+        <!-- Suppressed because we don't have so many imports that this is necessary -->
+        <!-- <module name="ImportControl"/> -->
+        <!-- Suppressed because we don't have so many imports that this is necessary -->
+        <!-- <module name="ImportOrder"/> -->
+        <module name="RedundantImport" />
+        <module name="UnusedImports" />
 
         <!-- Checks for Javadoc comments.                     -->
         <!-- See http://checkstyle.sourceforge.net/config_javadoc.html -->
         <module name="JavadocMethod">
             <property name="scope" value="package" />
-            <property name="allowMissingParamTags" value="true" />
+            <property name="allowMissingPropertyJavadoc" value="true" />
```

Comment:
Because getters and setters are generated by Intellij and Intellij doesn't think they need Javadoc. Additionally, it's a waste of time to write them because a good setter and a good getter are pretty obviously self-explanatory